source/GameInterface/GameInterface.csproj

```diff
@@ -34,6 +34,11 @@
     <WarningLevel>4</WarningLevel>
     <PlatformTarget>x64</PlatformTarget>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="Services\InputSystem\**" />
+    <EmbeddedResource Remove="Services\InputSystem\**" />
+    <None Remove="Services\InputSystem\**" />
+  </ItemGroup>
```

Comment:
What this do?

Reply:
no idea visual studio automatically did this?